tests/cluster/test_examples.sh

```
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# mock
+ctk() {
+  echo "$@"
+}
+
+test_cloud_cluster() {
+  source examples/shell/cloud_cluster.sh
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)